lib/src/register/cubits/register_cubit.dart

```diff
@@ -0,0 +1,45 @@
+import 'package:equatable/equatable.dart';
+import 'package:hydrated_bloc/hydrated_bloc.dart';
+
+import '../../../api/api.dart';
+import '../../../models/user.dart';
+import '../../../services/locator.dart';
+
+part 'register_state.dart';
+
+class RegisterCubit extends HydratedCubit<RegisterState> {
```

Comment:
This cubit has nothing to do with persisting the session (i.e. shouldn't be hydrated). Create a separate cubit named `AuthCubit` or `SessionCubit` to handle the flow.